crates/edr_napi/package.json

```diff
@@ -17,11 +17,15 @@
   "napi": {
     "name": "edr",
     "triples": {
+      "defaults": false,
```

Comment:
This change isn't strictly necessary for this PR, but while working on this I noticed that slang does it and I think it makes sense. This way we don't rely on napi-rs's defaults that might change without us noticing it.